### PR TITLE
fixed the width issue of nevermissanupdate container.

### PR DESCRIPTION
**Pull request by armedev**
# Pull Request Template

> Please provide all of the details below. Pull requests missing any of the information below will be closed!

**Sumarise what this PR fixes:** 

changed the width of the nevermissanupdate container in css.

**Fixes #:** #134 

# Checklist:
(Add an `x` between the brackets to check the items)

- [ `x`] I have sumarised what this PR fixes above
- [ `x`] I have provided the issue ID/URL above
- [ `x`] This PR fixes only the issue mentioned above (create seperate PRs for seperate issues)
- [  `x`] I was the first one to claim the issue or I created the issue

### CODE REVIEW

File: assets/css/neverMissAnUpdate.css [hunk 1]
```diff
@@ -5,27 +5,30 @@
...
-	width: 100vw;
-	height: auto;
-	display: flex;
-	justify-content: center;
-	background-color: #F4F7FE;
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  background-color: #f4f7fe;
```
Comment (armedev) [inline assets/css/neverMissAnUpdate.css:12]: changed the width to 100% rather than 100vw.

**Comment by armedev** [comment]
@MattCSmith and @brittneypostma please review the PR.

**Comment by MattCSmith** [comment]
Hey @armedev 
Apologies there is a conflict here I am unsure how to resolve. Any chance you could take a look and resolve it for me. 
Regards
Matt

**Comment by armedev** [comment]
@MattCSmith now you can review my pr.

**Comment by brittneypostma** [comment]
I found the pull request, I'm not sure how this is solving a responsive issue though. 100vw takes up the full viewport width, 100% will take up 100% of the parent container. I don't think this aligns with the quality pr that Digital Ocean expects for Hacktoberfest. I am going to close this one out.

**Comment by armedev** [comment]
> I found the pull request, I'm not sure how this is solving a responsive issue though. 100vw takes up the full viewport width, 100% will take up 100% of the parent container. I don't think this aligns with the quality pr that Digital Ocean expects for Hacktoberfest. I am going to close this one out.

actually my bad the issue was solved by setting X-scroll to hidden by @AdityaSaroj.

**Pull request closed without merging**